### PR TITLE
Change GuiDrawing constants to be full-alpha values

### PR DESCRIPTION
**Pull request by CalmBit**
Hey everyone,

Noticed while working with the basic GuiDrawing defaults that nothing was showing up - @unascribed gratefully lead me in the right direction towards noticing the defaults didn't render anything. This should fix that, giving them all full alpha values in order to render properly. If there's anything I missed or anything that I messed up, give me a holler!

### CODE REVIEW

File: src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java [hunk 1]
```diff
@@ -141,7 +141,9 @@ public static void rect(Fluid fluid, int left, int top, int width, int height, i
 	 * Draws a beveled, round rectangle that is substantially similar to default Minecraft UI panels.
 	 */
 	public static void drawGuiPanel(int x, int y, int width, int height) {
-		drawGuiPanel(x, y, width, height, 0x555555, 0xC6C6C6, 0xFFFFFF, 0x000000);
+		// For the purposes of Java's weird love-hate relationship with unsigned integers,
+		// -1 functions as 0xFFFFFFFF - full alpha + full white.
+		drawGuiPanel(x, y, width, height, 0xFF555555, 0xFFC6C6C6, -1, 0xFF000000);
```
Comment (exaskye) [inline src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java]: -1 is convenient shorthand in code, but for an API module like this I'd go for the longhand `0xFFFFFFFF`.

File: src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java [hunk 2]
```diff
@@ -141,7 +141,9 @@ public static void rect(Fluid fluid, int left, int top, int width, int height, i
 	 * Draws a beveled, round rectangle that is substantially similar to default Minecraft UI panels.
 	 */
 	public static void drawGuiPanel(int x, int y, int width, int height) {
-		drawGuiPanel(x, y, width, height, 0x555555, 0xC6C6C6, 0xFFFFFF, 0x000000);
+		// For the purposes of Java's weird love-hate relationship with unsigned integers,
```
Comment (falkreon) [inline src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java]: Ultimately unnecessary: 0xFFFFFFFF is a valid int constant.

**Review by falkreon — APPROVED** [review]
Huh. Yeah, this should get merged.

File: src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java (hunk 1 above)
Comment (CalmBit) [reply src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java]: IntelliJ _was_ yelling at me for that, but it stopped? I dunno...fixed it all the same, sorry about that

File: src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java (hunk 2 above)
Comment (CalmBit) [reply src/main/java/com/elytradev/concrete/inventory/gui/client/GuiDrawing.java]: See above: IntelliJ doesn't know _what_ it wants...fixed!

**Review by exaskye — APPROVED** [review]

**Pull request merged**